examples/asgilook/tests/test_images.py

```diff
@@ -0,0 +1,41 @@
+def test_list_images(client):
+    resp1 = client.simulate_get('/images')
+    assert resp1.status_code == 200
+    assert resp1.headers.get('X-ASGILook-Cache') == 'Miss'
+    assert resp1.json == []
+
+    resp2 = client.simulate_get('/images')
+    assert resp2.status_code == 200
+    assert resp2.headers.get('X-ASGILook-Cache') == 'Hit'
+    assert resp2.json == resp1.json
```

Comment:
Niptick: try also with an image?

Reply:
Main images are not cached, as described in the tutorial.
However, we should have a test asserting that thumbnails are not corrupted by caching. I'll check, and add if missing.

Reply:
Update: as explained above, images are not cached, only thumbnails are, which is what we can test.
To that end, I updated one of the tests in `test_thumbnails.py`.